metabase/metabase
# fix(sdk): fix defaultProps react error

Closes https://github.com/metabase/metabase/issues/48496

### Description

Removes these console errors

- `Modal: Support for defaultProps will be removed from function components in a future major release. Use JavaScript default parameters instead.`
- `ReactMarkdown: Support for defaultProps will be removed from function components in a future major release. Use JavaScript default parameters instead.`

### How to verify

- Renders the `InteractiveDashboard` component in either Vite or create-react-app
- Search for "defaultProps" in console
- Error should be gone

### Changelog for react-markdown version bump

8.0.7 - 2023-04-12

* [`c289176`](https://github.com/remarkjs/react-markdown/commit/c289176)
  Fix performance for keys
  by [**@wooorm**](https://github.com/wooorm)
  in [#738](https://github.com/remarkjs/react-markdown/pull/738)
* [`9034dbd`](https://github.com/remarkjs/react-markdown/commit/9034dbd)
  Fix types in syntax highlight example
  by [**@dlqqq**](https://github.com/dlqqq)
  in [#736](https://github.com/remarkjs/react-markdown/pull/736)

**Full Changelog**: <https://github.com/remarkjs/react-markdown/compare/8.0.6...8.0.7>

8.0.6 - 2023-03-20

* [`33ab015`](https://github.com/remarkjs/react-markdown/commit/33ab015)
  Update to TS 5\
  by [**@Methuselah96**](https://github.com/Methuselah96)
  in [#734](https://github.com/remarkjs/react-markdown/issues/734)

8.0.5 - 2023-01-17

* [`d640d40`](https://github.com/remarkjs/react-markdown/commit/d640d40)
  Update to use `node16` module resolution in `tsconfig.json`\
  by [**@ChristianMurphy**](https://github.com/ChristianMurphy)
  in [#723](https://github.com/remarkjs/react-markdown/pull/723)
* [`402fea3`](https://github.com/remarkjs/react-markdown/commit/402fea3)
  Fix typo in `plugins` deprecation message\
  by [**@marc2332**](https://github.com/marc2332)
  in [#719](https://github.com/remarkjs/react-markdown/pull/719)
* [`4f98f73`](https://github.com/remarkjs/react-markdown/commit/4f98f73)
  Remove deprecated and unneeded `defaultProps`\
  by [**@Lepozepo**](https://github.com/Lepozepo)
  in [#718](https://github.com/remarkjs/react-markdown/pull/718)
